commit d0243a9f9961f0afdc09b555821b88edb2488be9
Author: Stas Boukarev <stassats@gmail.com>
Date:   Wed Sep 30 02:46:31 2020 +0300

    Fix CPUID checking.
    
    cpu-has- is an index, not a mask. Have popcnt and no avx => bad time.

diff --git a/src/compiler/x86-64/parms.lisp b/src/compiler/x86-64/parms.lisp
index 5aeddff47..a9d2bef01 100644
--- a/src/compiler/x86-64/parms.lisp
+++ b/src/compiler/x86-64/parms.lisp
@@ -178,8 +178,8 @@ (defconstant linkage-table-entry-size 16)
 (defvar *binding-stack-pointer*)
 
 ;;; Bit indices into *CPU-FEATURE-BITS*
-(defconstant cpu-has-ymm-registers   1)
-(defconstant cpu-has-popcnt          2)
+(defconstant cpu-has-ymm-registers   0)
+(defconstant cpu-has-popcnt          1)
 
 (defconstant-eqx +static-symbols+
  `#(,@+common-static-symbols+
